src/components/chat.tsx

```
@@ -1,275 +1,779 @@
+                   setEditTitle(title);
+                }
+                // Always close on blur, whether saved, reverted, or unchanged
+                setIsRenamingAction(false);
+              }}
+              onKeyDown={(e) => {
+                 if (e.key === 'Escape') {
+                   setEditTitle(title);
+                   setIsRenamingAction(false);
+                 }
+              }}
+              autoFocus
```

Biome (1.9.4)

[error] Avoid the autoFocus attribute.

Unsafe fix: Remove the autoFocus attribute.


(lint/a11y/noAutofocus)